### DISCUSSION TITLE
v4 메세지가 멘션을 했는지 체크할수있을까요 ?

### DISCUSSION THREAD

**Discussion by difmwng3** [storycraft/node-kakao#404] [Q&A]
v4 로 

메세지가 

멘션이 된 메세지인이 체크를하고싶습니다.

**Comment by storycraft** [comment]
https://github.com/storycraft/node-kakao/blob/488d5a129a95ae810bbd56935db2f517ba09e7ac/src/talk/chat/index.ts#L161
chat 이벤트에서 주어지는 TalkChatData 객체 isMentioned메서드를 사용하세요.
